### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by oneiros-de** [jsevellec/cassandra-unit#339]
The last commit was on 2020-02-14, it has 62 open issues and 13 merge requests. 
And a main contributor (@jsevellec ) was last active on this platform on 2021-03-10.

**Comment by tiago-a** [comment]
Does any one know about a smiliar library for cassandra unit tests?

**Comment by Kiiv** [comment]
I think we need to use this now : https://www.mojohaus.org/cassandra-maven-plugin/

**Comment by jsevellec** [comment]
Yes — as of today, again. It genuinely was not when you asked in 2022, and the honest answer then
was the one you inferred from the commit log.

What landed today (#345):

- **Embedded Cassandra 3.11.5 → 5.0.8, and JDK 8 → 17.** That single pin was the root cause behind
  most of the tracker. Cassandra 3.11 is Java 8 only, and because this library starts Cassandra
  inside the test JVM, every consumer inherited it — which is why it would not run on JDK 11/17, or
  on Apple Silicon at all (there is no arm64 JDK 8).
- **A JUnit 5 extension**, so Jupiter and Spring Boot 3 suites have an entry point for the first
  time.
- **No known CVEs.** The dependency set went from roughly 60 published advisories across 15
  artifacts to zero across 96, verified against OSV.
- **CI**, which the project never had, and a working release path — publishing was pointed at
  `oss.sonatype.org`, which has since been shut down, so a release literally could not have been
  cut.
- **Documentation rewritten** and moved into the repository at
  https://github.com/jsevellec/cassandra-unit/tree/main/docs — the wiki was documenting classes and
  annotation attributes that do not exist (#231).

Being straight about what "alive" does and does not mean here:

- **5.0.0 is not released yet.** The code is on `main`; Maven Central still has 4.3.1.0 from January
  2020. Until that changes, nothing above helps you from a dependency declaration.
- **It is a deliberate break.** The command line tools, the shaded artifact and the XML/JSON/YAML
  dataset formats are gone, and JDK 17 is required —
  [the migration guide](https://github.com/jsevellec/cassandra-unit/blob/main/docs/migrating-from-4.md)
  lists all of it.
- **66 issues are still open.** Ten pull requests were closed today, most superseded by this work.

@tiago-a, @Kiiv — you were both right to look elsewhere in the meantime, and the alternatives are
still worth knowing about. [Testcontainers' Cassandra
module](https://java.testcontainers.org/modules/databases/cassandra/) is the closest equivalent and
avoids this project's central trade-off: because CassandraUnit runs the server in your test JVM, its
JDK is your JDK, and there is exactly one instance per JVM. Testcontainers runs a real node in
Docker with no such coupling. That constraint is permanent here, not a bug — Cassandra's static
state cannot be reset in-process — so it is documented rather than papered over.

Thanks for asking the question. It was a fair one, and it stayed open for four years, which rather
made your point.
